_schema.graphql

```diff
@@ -534,6 +534,8 @@ input CreateSubmissionMutationInput {
   editionSize: Int
   editionSizeFormatted: String
   height: String
+  locationAddress: String
+  locationAddress2: String
```

Comment:
What do you think about renaming this to `addressLine` to match the naming of Gravity's [UserAddress type](https://github.com/artsy/metaphysics/blob/10863e6753235b3286754d891db787471defbd24/_schemaV2.graphql#L20590).

```suggestion
  locationAddressLine1: String
  locationAddressLine2: String
```

Reply:
Sure, I'll do it. We were inspired by gravity's `address` and `address_2` but since MP converts it to `locationAddressLine` I think it's better to follow this convention

Reply:
we tried to match how it is in the Location type 
https://github.com/artsy/metaphysics/blob/585fff241d6291c65be31e31a8d6767f18c4a29f/_schemaV2.graphql#L11339-L11342

Reply:
Actually, both versions would work fine. My Collection stores the artwork location as [MyLocation](https://github.com/artsy/force/blob/726b38534a8ea68e80db4fa0f8e71bf45b7b96e6/data/schema.graphql#L14129), which uses `address` and `address2`.